### ISSUE TITLE
add aria-describedby to Languages List

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#167]
The Languages List should be reworked into a `<nav>` element containing a `<ul>` as well as `<div id=current-language hidden>`, for indicating which language in the list is the currently selected one.

The HTML for the Languages List should be extracted from its `render` method and moved into a separate HTML file, with the following:

```html
<nav class=languages-list>

  <ul>
  </ul>

  <div id=current-language hidden>Current Language</div>

</nav>
```
